examples/plugins/v2-local/devbox.json

```diff
@@ -11,10 +11,14 @@
     "scripts": {
       "run_test": [
         "test -n \"$PLUGIN1_INIT_HOOK\" || exit 1",
+        "test -n \"$PLUGIN1A_INIT_HOOK\" || exit 1",
         "test -n \"$PLUGIN1_ENV\" || exit 1",
+        "test -n \"$PLUGIN1A_ENV\" || exit 1",
         "if [ \"$PLUGIN1_ENV2\" != \"override\" ]; then exit 1; fi;",
         "devbox run plugin_1_script",
-        "hello"
+        "devbox run plugin_1A_script",
+        "hello",
+        "cowsay 'Hello, world!'"
```

Comment:
Would you mind adding some commentary to the lines in this file? Its unclear what scenarios are being demonstrated and vouched for by the various plugins. I can see some combination of:
1. relative, path:, plugin-with-json-file specified in `include` below.
2. plugins having different env-vars (aside from name, are there other differences?)
3. some plugin (which one?) introducing a `cowsay` package into the devbox project
4. plugins introducing scripts that can be `devbox run`

Since these are listed under `examples/`, we should make them user-friendly to understand because its reasonable that they'll browse the devbox projects inside `examples/` for inspiration and guidance on organizing their own projects.